src/sourmash/tax/tax_utils.py

```diff
@@ -296,7 +302,7 @@ def summarize_gather_at(rank, tax_assign, gather_results, *, skip_idents = [],
                     query_ani = containment_to_distance(fraction, ksize, scaled,
                                                         n_unique_kmers=qInfo.query_hashes, sequence_len_bp=qInfo.query_bp).ani
                 sres = SummarizedGatherResult(query_name, rank, fraction, lineage, query_md5,
-                                              query_filename, f_weighted_at_rank, bp_intersect_at_rank, query_ani)
+                                              query_filename, f_weighted_at_rank, bp_intersect_at_rank, query_ani, qInfo.total_weighted_hashes * scaled)
```

Comment:
`scaled` is not in the gather results prior to v4.4 (added at he same time as `ksize`) -- might be an issue with earlier gather results

Reply:
oh I see this is what you're checking for on like 478. Looks good!

Did we want to output the total weighted hashes in the csv summary too? Or just punt this until someone wants it?

Reply:
yep - handled elsewhere :)